manifest/idlharness.window.js

```diff
@@ -0,0 +1,17 @@
+// META: script=/resources/WebIDLParser.js
```

Comment:
Safe to delete this file entirely.

Reply:
Just to clarify, these files idlharness files only test WebIDL interfaces... as we don't actually define any of those, it's safe to delete this file.